index.html

```diff
@@ -3,9 +3,84 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Josh Glantz Codes</title>
+    <link rel="stylesheet" href="style/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;700&display=swap" rel="stylesheet">
+
 </head>
 <body>
-    
+
+    <header>
+        <h2>JOSH GLANTZ</h2>
+        <nav>
+            <a href="index.html">Home</a><!--
```

Comment:
If you give these a `display: inline-block` instead of their default `display: inline`, you don't have to do this dumb comment trick to get rid of the extra space.

Reply:
I knew there was another way! TY!

Reply:
Tried the following, but it inserted the spaces between links again.
```nav a {
    background: #EDEAE5;
    padding: 2px;
    color: black;
    display: inline-block;
}
```
resulting in:
![image](https://user-images.githubusercontent.com/65362632/87209590-492a2280-c2c7-11ea-83ce-eb7089260701.png)

Reply:
Sorry, It's been a while since I've needed to deal with that. When in React, its a non-issue. https://www.tutorialrepublic.com/faq/how-to-remove-the-space-between-inline-block-elements-in-css.php